### PR TITLE
add towncrier

### PR DESCRIPTION
**Pull request by kain88-de**
A little tool to help create changelog files.

### CODE REVIEW

**Comment by kain88-de** [comment]
Is `toml` not available on osx?

**Comment by kain88-de** [comment]
For `toml` see https://github.com/conda-forge/toml-feedstock/pull/1

**Comment by kain88-de** [comment]
This builds now on all platforms since the `toml` package was fixed.

File: recipes/towncrier/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - towncrier
+
+about:
+  home: https://github.com/hawkowl/towncrier
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Building newsfiles for your project'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (CJ-Wright) [inline recipes/towncrier/meta.yaml]: Please remove this comment

**Comment by kain88-de** [comment]
builds and files have been fixed.

**Comment by kain88-de** [comment]
Could someone have a look at this again. Thanks

File: recipes/towncrier/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+{% set version = "17.8.0" %}
+{% set sha256 = "8c0e5949851a9771a50d637b708322d73a4d4090fa354e5cb24a4f7464e81d47" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (CJ-Wright) [inline recipes/towncrier/meta.yaml]: Can this be `noarch`?

**Comment by CJ-Wright** [comment]
This is noarch so we can merge without travis.

**Pull request merged**